model/metrics/container.yaml

```diff
@@ -11,7 +11,21 @@ groups:
     unit: "s"
     attributes:
       - ref: container.cpu.state
-        brief: "The CPU state for this data point. A container SHOULD be characterized _either_ by data points with no `state` labels, _or only_ data points with `state` labels."
+        brief: "The CPU state for this data point. A container's cpu time SHOULD be characterized _either_ by data points with no `state` labels, _or only_ data points with `state` labels."
+        requirement_level: opt_in
+
+  - id: metric.container.cpu.usage
+    type: metric
+    metric_name: container.cpu.usage
+    stability: experimental
+    brief: "Container's CPU usage, measured in cpus. Range from 0 to the number of allocatable cpus"
+    note: >
+      CPU usage of the specific container on all available CPU cores, averaged over the sample window
+    instrument: gauge
+    unit: "{cpu}"
+    attributes:
+      - ref: container.cpu.state
+        brief: "The CPU state for this data point. A container's cpu usage SHOULD be characterized _either_ by data points with no `state` labels, _or only_ data points with `state` labels."
         requirement_level: opt_in
```

Comment:
Why not `recommended`? Having one dimension with 3 possible values on the metric should be fine.

Reply:
As explained also at https://github.com/open-telemetry/semantic-conventions/pull/1128#discussion_r1634449851, `kubeletstats` cannot distinguish the state/mode while the `dockerstats` can. That's why we marked it as opt-in when we [introduced that](https://github.com/open-telemetry/semantic-conventions/pull/282).

Reply:
You could also use conditionally required (where condition is dockerstats)

Reply:
Thank's for the suggestion. Added at [9bc986b](https://github.com/open-telemetry/semantic-conventions/pull/1128/commits/9bc986b361db6144c768d53590fed1ac86a757f3). For some reason it has no impact on the MD table though 🤔 (even after running `make fix` or `make table-generation `).

Reply:
We spoke on slack, @lquerel will investigate what looks like a bug in the tooling.

Reply:
I think we can move this forward by changing the requirement level in the `container.cpu.time` as well, which is something needed anyways. And the issue in the tooling can be investigated independently.

Reply:
As I mentioned in the other comment, there seem to be no issue on tooling side, we just need to use proper group id (related to cpu.usage and not time)